docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>Accuracy should be passed to the metrics parameter of model.compile().</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
What do you mean by this? Accuracy is used as the metric in `model.compile()`

Reply:
<p>My mistake</p>